clusters/build-clusters/02_cluster/prow/03_deployment/admin_hook_rbac.yaml

```diff
@@ -0,0 +1,88 @@
+kind: List
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: hook
+    namespace: ci
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: hook
+    namespace: ci
+  rules:
+    - apiGroups:
+        - ""
+      resources:
+        - configmaps
+      verbs:
+        - get
+        - update
+        - create
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: hook
+    namespace: ci
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: hook
+  subjects:
+  - kind: ServiceAccount
+    name: hook
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: hook
+    namespace: ci-stg
```

Comment:
I think we decided not to use the `ci-stg` namespace anymore

Reply:
It is still used in secret-mirror tools.
Maybe do it later to remove it on all of cluster?

Reply:
Let's not create it, it's deprecated.

Reply:
Done.